src/pom.xml

```diff
@@ -135,7 +135,7 @@
         <spring.version>4.3.11.RELEASE</spring.version>
         <spring-security.version>4.2.3.RELEASE</spring-security.version>
         <!-- TODO: Raise log4j to version 2 -->
```

Comment:
Very, very minor: This TODO is done by your change, isn't it? So it can be removed.

Reply:
True, removed it.